src/structures/AnonymousGuild.js

```diff
@@ -68,22 +68,20 @@ class AnonymousGuild extends BaseGuild {
 
   /**
    * The URL to this guild's banner.
-   * @param {ImageURLOptions} [options={}] Options for the Image URL
+   * @param {StaticImageURLOptions} [options={}] Options for the Image URL
```

Comment:
Banners can be animated, right?

Reply:
That's not released yet, see discord/discord-api-docs#3948